backend/app/ml_routes.py

```
@@ -0,0 +1,194 @@
+from fastapi import APIRouter, HTTPException, UploadFile, File, Form
+from fastapi.responses import JSONResponse
+from .ml_service import MedicationComparisonService
+from .user_service import UserService
+from .firebase_config import get_firestore_client
+from typing import Optional
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)